Python packages set: patch updates

FRidh opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#chap-reviewing-contributions
-->

###### Motivation for this change

```
$ maintainers/scripts/update-python-libraries --use-pkgs-prefix --commit --target=patch pkgs/development/python-modules/**/*.nix
```

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

drewrisinger committed:
pythonPackages.pandas: cleanup

* Sort imports
* add changelog

drewrisinger committed:
pythonPackages.pandas: convert to pytestCheckHook

nbke committed:
pythonPackages.numpy: set license to bsd3

FRidh committed:
python3Packages.apispec: 3.3.1 -> 3.3.2

FRidh committed:
python3Packages.GitPython: 3.1.7 -> 3.1.11

FRidh committed:
python3Packages.aiohttp: 3.6.2 -> 3.6.3

FRidh committed:
python3Packages.amply: 0.1.2 -> 0.1.4

FRidh committed:
python3Packages.amazon_kclpy: 1.5.0 -> 1.5.1

FRidh committed:
python3Packages.alembic: 1.4.2 -> 1.4.3

FRidh committed:
python3Packages.adal: 1.2.4 -> 1.2.5

FRidh committed:
python3Packages.bokeh: 2.2.1 -> 2.2.3

FRidh committed:
python3Packages.astropy: 4.0.1.post1 -> 4.0.3

FRidh committed:
python3Packages.bayespy: 0.5.19 -> 0.5.20

FRidh committed:
python3Packages.awkward1: 0.2.35 -> 0.2.38

FRidh committed:
python3Packages.aws-adfs: 1.24.4 -> 1.24.5

FRidh committed:
python3Packages.atlassian-python-api: 1.17.5 -> 1.17.6

FRidh committed:
python3Packages.beautifulsoup4: 4.9.1 -> 4.9.3

FRidh committed:
python3Packages.cornice: 5.0.2 -> 5.0.3

FRidh committed:
python3Packages.djangorestframework: 3.11.1 -> 3.11.2

FRidh committed:
python3Packages.configparser: 5.0.0 -> 5.0.1

FRidh committed:
python3Packages.cvxpy: 1.1.6 -> 1.1.7

FRidh committed:
python3Packages.boto3: 1.15.9 -> 1.15.18

FRidh committed:
python3Packages.cffi: 1.14.2 -> 1.14.3

FRidh committed:
python3Packages.colorama: 0.4.3 -> 0.4.4

FRidh committed:
python3Packages.browser-cookie3: 0.11.1 -> 0.11.3

FRidh committed:
python3Packages.botocore: 1.18.9 -> 1.18.18

FRidh committed:
python3Packages.codecov: 2.1.9 -> 2.1.10

FRidh committed:
python3Packages.croniter: 0.3.34 -> 0.3.35

FRidh committed:
python3Packages.django-reversion: 3.0.7 -> 3.0.8

FRidh committed:
python3Packages.deform: 2.0.10 -> 2.0.14

FRidh committed:
python3Packages.django-simple-captcha: 0.5.12 -> 0.5.13

FRidh committed:
python3Packages.google-cloud-dataproc: 2.0.0 -> 2.0.2

FRidh committed:
python3Packages.google-cloud-core: 1.4.1 -> 1.4.3

FRidh committed:
python3Packages.dulwich: 0.20.5 -> 0.20.6

FRidh committed:
python3Packages.dotnetcore2: 2.1.14 -> 2.1.17

FRidh committed:
python3Packages.Flask-Cors: 3.0.8 -> 3.0.9

FRidh committed:
python3Packages.google-cloud-bigquery-datatransfer: 1.1.0 -> 1.1.1

FRidh committed:
python3Packages.greenlet: 0.4.16 -> 0.4.17

FRidh committed:
python3Packages.eventlet: 0.28.0 -> 0.28.1

FRidh committed:
python3Packages.green: 3.2.1 -> 3.2.3

FRidh committed:
python3Packages.flake8: 3.8.3 -> 3.8.4

FRidh committed:
python3Packages.google-cloud-datastore: 1.15.0 -> 1.15.3

FRidh committed:
python3Packages.google-api-core: 1.22.1 -> 1.22.4

FRidh committed:
python3Packages.gym: 0.17.2 -> 0.17.3

FRidh committed:
python3Packages.Faker: 4.1.2 -> 4.1.8

FRidh committed:
python3Packages.Fiona: 1.8.13.post1 -> 1.8.17

FRidh committed:
python3Packages.nbformat: 5.0.7 -> 5.0.8

FRidh committed:
python3Packages.matplotlib: 3.3.1 -> 3.3.2

FRidh committed:
python3Packages.msgpack-numpy: 0.4.6.1 -> 0.4.7.1

FRidh committed:
python3Packages.labelbox: 2.4.4 -> 2.4.6

FRidh committed:
python3Packages.ledgerblue: 0.1.31 -> 0.1.32

FRidh committed:
python3Packages.hmmlearn: 0.2.3 -> 0.2.4

FRidh committed:
python3Packages.kaggle: 1.5.6 -> 1.5.9

FRidh committed:
python3Packages.mecab-python3: 1.0.1 -> 1.0.2

FRidh committed:
python3Packages.msal: 1.5.0 -> 1.5.1

FRidh committed:
python3Packages.ipyvue: 1.4.0 -> 1.4.1

FRidh committed:
python3Packages.ipdb: 0.13.3 -> 0.13.4

FRidh committed:
python3Packages.jupyterlab: 2.2.8 -> 2.2.9

FRidh committed:
python3Packages.nest_asyncio: 1.4.0 -> 1.4.1

FRidh committed:
python3Packages.maxminddb: 2.0.2 -> 2.0.3

FRidh committed:
python3Packages.munkres: 1.1.2 -> 1.1.4

FRidh committed:
python3Packages.iso8601: 0.1.12 -> 0.1.13

FRidh committed:
python3Packages.passlib: 1.7.2 -> 1.7.4

FRidh committed:
python3Packages.prompt_toolkit: 3.0.6 -> 3.0.8

FRidh committed:
python3Packages.parver: 0.3.0 -> 0.3.1

FRidh committed:
python3Packages.nunavut: 0.3.6 -> 0.3.9

FRidh committed:
python3Packages.packet-python: 1.43.0 -> 1.43.1

FRidh committed:
python3Packages.openwrt-luci-rpc: 1.1.3 -> 1.1.6

FRidh committed:
python3Packages.numpy-stl: 2.11.2 -> 2.11.3

FRidh committed:
python3Packages.numba: 0.51.1 -> 0.51.2

FRidh committed:
python3Packages.PasteDeploy: 2.1.0 -> 2.1.1

FRidh committed:
python3Packages.pandas: 1.1.1 -> 1.1.3

FRidh committed:
python3Packages.proto-plus: 1.10.1 -> 1.10.2

FRidh committed:
python3Packages.numpy: 1.19.1 -> 1.19.2

FRidh committed:
python3Packages.pandocfilters: 1.4.2 -> 1.4.3

FRidh committed:
python3Packages.pg8000: 1.16.5 -> 1.16.6

FRidh committed:
python3Packages.psutil: 5.7.2 -> 5.7.3

FRidh committed:
python3Packages.psd-tools: 1.9.15 -> 1.9.16

FRidh committed:
python3Packages.PyChromecast: 7.5.0 -> 7.5.1

FRidh committed:
python3Packages.py4j: 0.10.9 -> 0.10.9.1

FRidh committed:
python3Packages.pybase64: 1.0.1 -> 1.0.2

FRidh committed:
python3Packages.psycopg2: 2.8.5 -> 2.8.6

FRidh committed:
python3Packages.pyhomematic: 0.1.68 -> 0.1.70

FRidh committed:
python3Packages.pyct: 0.4.6 -> 0.4.8

FRidh committed:
python3Packages.pybotvac: 0.0.17 -> 0.0.18

FRidh committed:
python3Packages.pybids: 0.12.1 -> 0.12.3

FRidh committed:
python3Packages.pycurl: 7.43.0.5 -> 7.43.0.6

FRidh committed:
python3Packages.pyexcel-xls: 0.6.0 -> 0.6.1

FRidh committed:
python3Packages.PyMuPDF: 1.17.6 -> 1.17.7

FRidh committed:
python3Packages.pytest-testmon: 1.0.2 -> 1.0.3

FRidh committed:
python3Packages.pytest-flakes: 4.0.1 -> 4.0.2

FRidh committed:
python3Packages.pytest-black: 0.3.11 -> 0.3.12

FRidh committed:
python3Packages.pyrsistent: 0.16.0 -> 0.16.1

FRidh committed:
python3Packages.pyopencl: 2020.2.1 -> 2020.2.2

FRidh committed:
python3Packages.pytest-quickcheck: 0.8.4 -> 0.8.5

FRidh committed:
python3Packages.pyotp: 2.4.0 -> 2.4.1

FRidh committed:
python3Packages.pyperclip: 1.8.0 -> 1.8.1

FRidh committed:
python3Packages.python-rapidjson: 0.9.1 -> 0.9.3

FRidh committed:
python3Packages.python-hosts: 1.0.0 -> 1.0.1

FRidh committed:
python3Packages.pywbem: 1.0.2 -> 1.0.3

FRidh committed:
python3Packages.pytest-sanic: 1.6.1 -> 1.6.2

FRidh committed:
python3Packages.pytools: 2020.4 -> 2020.4.3

FRidh committed:
python3Packages.pyspread: 1.99.2 -> 1.99.4

FRidh committed:
python3Packages.requests-aws4auth: 1.0 -> 1.0.1

FRidh committed:
python3Packages.robotframework: 3.2.1 -> 3.2.2

FRidh committed:
python3Packages.slither-analyzer: 0.6.12 -> 0.6.13

FRidh committed:
python3Packages.reportlab: 3.5.48 -> 3.5.53

FRidh committed:
python3Packages.ruamel.yaml: 0.16.10 -> 0.16.12

FRidh committed:
python3Packages.sphinxcontrib-plantuml: 0.18 -> 0.18.1

FRidh committed:
python3Packages.scour: 0.38 -> 0.38.1.post1

FRidh committed:
python3Packages.scipy: 1.5.2 -> 1.5.3

FRidh committed:
python3Packages.SQLAlchemy: 1.3.19 -> 1.3.20

FRidh committed:
python3Packages.sh: 1.14.0 -> 1.14.1

FRidh committed:
python3Packages.sqlmap: 1.4.9 -> 1.4.10

FRidh committed:
python3Packages.sounddevice: 0.4.0 -> 0.4.1

FRidh committed:
python3Packages.screeninfo: 0.6.5 -> 0.6.6

FRidh committed:
python3Packages.ruamel.ordereddict: 0.4.14 -> 0.4.15

FRidh committed:
python3Packages.testfixtures: 6.14.1 -> 6.14.2

FRidh committed:
python3Packages.srptools: 1.0.0 -> 1.0.1

FRidh committed:
python3Packages.transitions: 0.8.2 -> 0.8.4

FRidh committed:
python3Packages.txdbus: 1.1.1 -> 1.1.2

FRidh committed:
python3Packages.tasklib: 2.2.0 -> 2.2.1

FRidh committed:
python3Packages.u-msgpack-python: 2.7.0 -> 2.7.1

FRidh committed:
python3Packages.streamz: 0.5.5 -> 0.5.6

FRidh committed:
python3Packages.virtualenv: 20.0.31 -> 20.0.35

FRidh committed:
python3Packages.vcrpy: 4.1.0 -> 4.1.1

FRidh committed:
python3Packages.xhtml2pdf: 0.2.4 -> 0.2.5

FRidh committed:
python3Packages.urllib3: 1.25.10 -> 1.25.11

FRidh committed:
python3Packages.urwid: 2.1.1 -> 2.1.2

FRidh committed:
python3Packages.textacy: 0.10.0 -> 0.10.1

FRidh committed:
python3Packages.zfec: 1.5.3 -> 1.5.4

FRidh committed:
python3Packages.zeroconf: 0.28.5 -> 0.28.6

FRidh committed:
python3Packages.zha-quirks: 0.0.43 -> 0.0.45

FRidh committed:
python3Packages.zopfli: 0.1.6 -> 0.1.7

FRidh committed:
python3Packages.zope.interface: 5.1.0 -> 5.1.2

SuperSandro2000 commented on pkgs/development/python-modules/boto3/default.nix:
```diff
@@ -13,11 +13,11 @@
 buildPythonPackage rec {
   pname =  "boto3";
-  version = "1.15.9"; # N.B: if you change this, change botocore too
+  version = "1.15.18"; # N.B: if you change this, change botocore too
```
```suggestion
  version = "1.15.18"; # N.B: if you change this, change boto3 and awscli to a matching version
```

SuperSandro2000 commented:
Did you use a script to do this? Can I find it somewhere?

FRidh commented:
@SuperSandro2000 
```
$ maintainers/scripts/update-python-libraries --use-pkgs-prefix --commit --target=patch pkgs/development/python-modules/**/*.nix
```

mweinelt commented:
The zipgy updates collide with #101557 and I'd like the maintainer to work on test integration in that pull request. They also require an update to home-assistants `component-packages.nix`, so I'd appreciate if you could drop them here.

mweinelt commented:
```
builder for '/nix/store/zg8jl91486n45sz196s2z3zmxs74g36m-python3.8-amply-0.1.4.drv' failed with exit code 1; last 10 log lines:
  patching script interpreter paths in /nix/store/c81cvqg9yjzhvf3ff5b25kgxwzc0g8l2-python3.8-amply-0.1.4
  checking for references to /build/ in /nix/store/c81cvqg9yjzhvf3ff5b25kgxwzc0g8l2-python3.8-amply-0.1.4...
  Executing pythonRemoveTestsDir
  Finished executing pythonRemoveTestsDir
  running install tests
  Traceback (most recent call last):
    File "tests/test_amply.py", line 4, in <module>
      import pytest
  ModuleNotFoundError: No module named 'pytest'
  builder for '/nix/store/zg8jl91486n45sz196s2z3zmxs74g36m-python3.8-amply-0.1.4.drv' failed with exit code 1
```

https://github.com/willu47/amply/blame/v0.1.4/tests/test_amply.py#L4

FRidh committed:
python3Packages.aioconsole: 0.3.0 -> 0.3.1

FRidh committed:
python3Packages.browser-cookie3: 0.11.3 -> 0.11.4

FRidh committed:
python3Packages.atlassian-python-api: 1.17.6 -> 1.17.7

FRidh committed:
python3Packages.dash_renderer: 1.8.2 -> 1.8.3

FRidh committed:
python3Packages.blis: 0.7.1 -> 0.7.2

FRidh committed:
python3Packages.croniter: 0.3.35 -> 0.3.36

FRidh committed:
python3Packages.chalice: 1.21.3 -> 1.21.4

FRidh committed:
python3Packages.Genshi: 0.7.3 -> 0.7.4

FRidh committed:
python3Packages.identify: 1.5.6 -> 1.5.9

FRidh committed:
python3Packages.djangoql: 0.14.0 -> 0.14.2

FRidh committed:
python3Packages.green: 3.2.3 -> 3.2.4

FRidh committed:
python3Packages.Django: 2.2.16 -> 2.2.17

FRidh committed:
python3Packages.Django: 3.1.2 -> 3.1.3

FRidh committed:
python3Packages.google-auth-oauthlib: 0.4.1 -> 0.4.2

FRidh committed:
python3Packages.dulwich: 0.20.6 -> 0.20.11

FRidh committed:
python3Packages.deepmerge: 0.1.0 -> 0.1.1

FRidh committed:
python3Packages.hidapi: 0.10.0 -> 0.10.0.post1

FRidh committed:
python3Packages.django-q: 1.3.3 -> 1.3.4

FRidh committed:
python3Packages.distributed: 2.30.0 -> 2.30.1

FRidh committed:
python3Packages.ECPy: 1.2.3 -> 1.2.5

FRidh committed:
python3Packages.libtmux: 0.8.3 -> 0.8.5

FRidh committed:
python3Packages.pymavlink: 2.4.11 -> 2.4.12

FRidh committed:
python3Packages.pycoin: 0.90.20200809 -> 0.90.20201031

FRidh committed:
python3Packages.pycryptodome: 3.9.8 -> 3.9.9

FRidh committed:
python3Packages.preshed: 3.0.2 -> 3.0.3

FRidh committed:
python3Packages.pandas: 1.1.3 -> 1.1.4

FRidh committed:
python3Packages.pyexcel-io: 0.6.3 -> 0.6.4

FRidh committed:
python3Packages.pycryptodomex: 3.9.8 -> 3.9.9

FRidh committed:
python3Packages.murmurhash: 1.0.2 -> 1.0.4

FRidh committed:
python3Packages.notify_py: 0.2.3 -> 0.2.4

FRidh committed:
python3Packages.ledgerblue: 0.1.32 -> 0.1.34

FRidh committed:
python3Packages.numpy: 1.19.2 -> 1.19.4

FRidh committed:
python3Packages.junos-eznc: 2.5.3 -> 2.5.4

FRidh committed:
python3Packages.jupyterlab_git: 0.22.2 -> 0.22.3

FRidh committed:
python3Packages.nest_asyncio: 1.4.1 -> 1.4.2

FRidh committed:
python3Packages.phonenumbers: 8.12.11 -> 8.12.12

FRidh committed:
python3Packages.thinc: 7.4.1 -> 7.4.2

FRidh committed:
python3Packages.reportlab: 3.5.53 -> 3.5.55

FRidh committed:
python3Packages.statsmodels: 0.12.0 -> 0.12.1

FRidh committed:
python3Packages.sshtunnel: 0.2.1 -> 0.2.2

FRidh committed:
python3Packages.tubeup: 0.0.19 -> 0.0.20

FRidh committed:
python3Packages.scp: 0.13.2 -> 0.13.3

FRidh committed:
python3Packages.setuptools-rust: 0.11.3 -> 0.11.4

FRidh committed:
python3Packages.sqlmap: 1.4.10 -> 1.4.11

FRidh committed:
python3Packages.tempora: 4.0.0 -> 4.0.1

FRidh committed:
python3Packages.pytest: 6.1.1 -> 6.1.2

FRidh committed:
python3Packages.snowflake-connector-python: 2.3.4 -> 2.3.5

FRidh committed:
python3Packages.toml: 0.10.1 -> 0.10.2

FRidh committed:
python3Packages.periodictable: 1.5.2 -> 1.5.3

FRidh committed:
python3Packages.scipy: 1.5.3 -> 1.5.4

FRidh committed:
python3.pkgs.genshi: fix build

jonringer committed:
python3Packages.browser-cookie3: fix deps

jonringer commented:
fridh, your pip reproducible patch isn't included in here. many packages are failing

jonringer committed:
python3Packages.dash_core_components: 1.12.1 -> 1.13.0

jonringer committed:
python3Packages.dash: 1.16.2 -> 1.17.0

jonringer committed:
python3Packages.dash_table: 4.10.1 -> 4.11.0

jonringer committed:
python3Packages.update_checker: disable py2

```
builder for '/nix/store/2qwl0rkdm2gx2byhpmw7lxyg5rprwqqk-python2.7-update_checker-0.18.0.drv' failed with exit code 1; last 10 log lines:
  no configure script, doing nothing
  building
  Executing setuptoolsBuildPhase
  Traceback (most recent call last):
    File "nix_run_setup", line 8, in <module>
      exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
    File "setup.py", line 9
      with open(f"{MODULE_NAME}.py") as fp:
                                  ^
  SyntaxError: invalid syntax
```

jonringer committed:
python3Packages.sphinxcontrib-openapi: disable py2

```
builder for '/nix/store/jz41wvwvkc6q55972ysdajm6k3vnv6r8-python2.7-sphinxcontrib-openapi-0.7.0.drv' failed with exit code 1; last 10 log lines:
  configuring
  no configure script, doing nothing
  building
  Executing setuptoolsBuildPhase
  Traceback (most recent call last):
    File "nix_run_setup", line 8, in <module>
      exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
    File "setup.py", line 10, in <module>
      with open(os.path.join(here, "README.rst"), "r", encoding="utf-8") as f:
  TypeError: 'encoding' is an invalid keyword argument for this function
```

jonringer committed:
python3Packages.pytest-flakes: disable py2

jonringer committed:
Revert "python: patator: 0.7 -> 0.9"

This reverts commit 5c1c22ad8cfde07d6d6c45cd2ea5f3508842116f.

jonringer committed:
python3Packages.flask-cores: fix tests

jonringer committed:
python3Packages.amply: fix tests

jonringer committed:
python3Packages.google-auth-oauthlib: disable python<3.6

jonringer committed:
python3Packages.pyct: disable py2 tests

jonringer committed:
python3Packages.requests_aws4auth: disable tests, no longer in pyip

jonringer committed:
python3Packages.munkres: fix tests

jonringer committed:
python3Packages.py-multihash: mark broken, unmaintained

jonringer committed:
python3Packages.premailer: disable for py2

jonringer committed:
python3Packages.pushover-complete: disable for py2

jonringer committed:
python3Packages.zope_copy: disable tests for py2

jonringer committed:
python3Packages.iterm2: make darwin only

requires pyobjc package, which is only available for darwin

jonringer committed:
python3Packages.nbconflux: disable for py2

jonringer committed:
python3Packages.jupytext: fix deps

jonringer committed:
python3Packages.pycurl: disable for py2

jonringer committed:
python37Packages.sphinxcontrib-spelling: fix deps

jonringer committed:
python3Packages.fiona: fix deps

jonringer commented:
@fridh I rebased this on top of the latest to bringing your fixes for builds affected by the pip reproducibility patch.

Unfortunately, I'm still affected by https://github.com/xianyi/OpenBLAS/issues/2970 and unable to build numpy. So, I'll need to wait for hydra to build it for me to progress much further

jonringer committed:
python3Packages.pandas: fix tests

jonringer committed:
python3Packages.rubymarshal: disable tests

jonringer committed:
python2Packages.screeninfo: disable python2

jonringer committed:
python3Packages.xdis: 5.0.4 -> 5.0.5

jonringer committed:
pythonPackages.ipdb: disable python2

jonringer committed:
libsoup: fix build

jonringer committed:
python3Packages.periodictable: fix tests

jonringer committed:
python3Packages.deform: fix tests

jonringer committed:
python3Packages.ipdb: add missing test deps

jonringer committed:
python3Packages.scramp: init at 1.2.0

jonringer committed:
python3Packages.pg8000: add missing deps

jonringer committed:
python3Packages.google_auth: 1.20.1 -> 1.23.0

jonringer commented:
this is probably more stable than master at this point

raboof commented:
The pytest-testmon 1.0.2 -> 1.0.3 included in this change appears to fail consistently.

I'm no python programmer, but indeed looking at the [PyPI package](https://pypi.org/project/pytest-testmon/#files), `test/test_core.py:131` expects `filenames_fingerprints` to contain 5 elements, but the query at `testmon/testmon_core.py:267` only selects 4.

I don't see any tests in the source repo at https://github.com/tarpas/pytest-testmon, though, so it's unclear to me where these tests actually come from...

raboof commented:
> The pytest-testmon 1.0.2 -> 1.0.3 included in this change appears to fail consistently.

It seems the tests are expecting master instead of 1.0.3 (https://github.com/tarpas/pytest-testmon/issues/158), let's skip them for now (https://github.com/NixOS/nixpkgs/pull/103786)
